exporter.go

```diff
@@ -239,6 +242,15 @@ var (
 	listenAddr = flag.String("web.listen-address",
 		defaultListenAddress,
 		"An address or unix domain socket path to listen on for web interface and telemetry. The default value can be overwritten by LISTEN_ADDRESS environment variable.")
+	securedMetrics = flag.Bool("web.secured-metrics",
+		defaultSecuredMetrics,
+		"Expose metrics using https. The default value can be overwritten by SECURED_METRICS variable.")
+	sslServerCert = flag.String("web.ssl-server-cert",
+		defaultSslServerCert,
+		"Certificate for the nginx-exporter metrics server(when web.secured-metrics=true). The default value can be overwritten by SSL_SERVER_CERT variable.")
```

Comment:
This is for for consistency with the existing flags (`nginx.ssl-ca-cert`, `nginx.ssl-client-cert`, ...) 

```suggestion
		"Path to the PEM encoded certificate for the nginx-exporter metrics server (when web.secured-metrics=true). The default value can be overwritten by SSL_SERVER_CERT variable.")
```
Could you also update the README accordingly?

Reply:
Fixed

Reply:
done also @ README, thanks!